lib/ClangImporter/ImportDecl.cpp

```diff
@@ -1087,12 +1087,20 @@ createValueConstructor(ClangImporter::Implementation &Impl,
   // Construct the set of parameters from the list of members.
   SmallVector<ParamDecl *, 8> valueParameters;
   for (auto var : members) {
-    // TODO create value constructor with indirect fields instead of the
-    // generated __Anonymous_field.
-    if (var->hasClangNode() && isa<clang::IndirectFieldDecl>(var->getClangDecl()))
-      continue;
+    bool generateParamName = wantCtorParamNames;
+
+    if (var->hasClangNode()) {
```

Comment:
Nitpick: It seems like we should still honor `wantCtorParamNames` being `false`, even if no callers do this now with imported structs.